frontend/src/components/AdminComponent.js

```diff
@@ -0,0 +1,30 @@
+import React, { useContext, useEffect } from "react";
+import TableComponent from "./TableComponent";
+import AddModal from "./AddModal";
+import HeaderComponent from "./HeaderComponent";
+import { userContext } from "../context/userContext";
+
+const AdminComponent = () => {
+  const { users, getUsers, addUser, deleteUser } = useContext(userContext);
+
+  useEffect(() => {
+    getUsers();
+  }, []);
```

Comment:
please check whether this function calling the getUser() continuously.

Reply:
Yes it worked when i was using the json-server. After integrating with the backend there are problems and when coming upto that point i get an error saying you are not authenticated.